static/js/multiselectfilter.js

```diff
@@ -0,0 +1,126 @@
+window.onload = function () {
```

Comment:
use same weird model for all GDS modules, and same weird JavaScript conventions which I can only guess has been proved to work on really old browsers